Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -142,6 +153,7 @@ def tap_new
         safe_system "git", "init"
         safe_system "git", "add", "--all"
         safe_system "git", "commit", "-m", "Create #{tap} tap"
+        safe_system "git", "branch", "-m", branch
```

Comment:
Maybe better specify `-b` flag for `git init` directly?

Reply:
I had it this way initially but tests failed. I think the `-b` flag is a new feature that isn't available in the system versions of `git`